### PR TITLE
Added release notes for 3.2 RC

### PR DESCRIPTION
**Pull request by csharpfritz**
WIP

### CODE REVIEW

**Comment by csharpfritz** [comment]
New Features at the top list items that are common to both environments

File: NuGet.Docs/Release-Notes/NuGet-2.8.7.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # NuGet 2.8.7 Release Notes
 
-[NuGet 2.8.5 Release Notes](nuget-2.8.5) | [NuGet 2.8.7 Release Notes](nuget-2.8.7)
+[NuGet 2.8.5 Release Notes](nuget-2.8.5) | [NuGet 3.0 Preview Release Notes](nuget-3.0-preview)
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-2.8.7.md]: this looks incorrect. The text below still talks about nuget 2.8.7

File: NuGet.Docs/Release-Notes/NuGet-3.2-RC.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
+# NuGet 3.2 RC Release Notes
+
+[NuGet 3.1.1 Release Notes](nuget-3.1.1)
+
+NuGet 3.2 release candidate was released September 2, 2015 as a collection of improvements and fixes for the 3.1.1 release.  Also, these are the first releases that are published first to the new dist.nuget.org repository.
+
+## New Features
+
+* Projects that live in the same folder can now have different project.json files in that folder specific to each project.  For each project, name the project.json file {ProjectName}.project.json and NuGet will properly reference and use that content for each project appropriately. ([1102](https://github.com/NuGet/Home/issues/1102))
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.2-RC.md]: this is depending one a feature from a visual studio update to be released with win 10 tools

File: NuGet.Docs/Release-Notes/NuGet-3.2-RC.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+[NuGet 3.1.1 Release Notes](nuget-3.1.1)
+
+NuGet 3.2 release candidate was released September 2, 2015 as a collection of improvements and fixes for the 3.1.1 release.  Also, these are the first releases that are published first to the new dist.nuget.org repository.
+
+## New Features
+
+* Projects that live in the same folder can now have different project.json files in that folder specific to each project.  For each project, name the project.json file {ProjectName}.project.json and NuGet will properly reference and use that content for each project appropriately. ([1102](https://github.com/NuGet/Home/issues/1102)) 
+* NuGet.config now supports a globalPackagesFolder as a relative path - [1062](https://github.com/NuGet/Home/issues/1062)
+
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.2-RC.md]: Restore context menu is another small new feature

File: NuGet.Docs/Release-Notes/NuGet-3.2-RC.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+### New Features
+
+* A new context menu item was added to the Solution Explorer on the solution node that allows packages to be restored without building the solution ([1274](https://github.com/NuGet/Home/issues/1274)).
+
+![New 'Restore Packages' Context Menu Item](images/NuGet-3.2/newContextMenu.png)
+
+### Updates and Fixes
+
+The security fixes for authenticated feeds were rolled up and addressed in the extension as well.  The following authentication items were also addressed in the extension:
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.2-RC.md]: Don't use the term security. Use authentication instead.

**Pull request merged**